www/pages/docs/data.md

```diff
@@ -0,0 +1,321 @@
+
+As an example, if you are developing a blog site, like in our [Getting Started](/getting-started/) guide, having to list a couple blogs posts by hand isn't so bad.
+
+```render html
+<ul>
+  <li><a href="/blog/2019/first-post.md">First Post</li></a>
+  <li><a href="/blog/2019/second-post.md">Second Post</li></a>
+</ul>
+```
+
+But what happens over time, when that list grows to 10, 50, 100+ posts?  Imagine maintaining that list each time, over and over again?  Not only that, but wouldn't it be great to also be able to sort, search, filter, and organize those posts to make them easier for users to navigate and find?  Even better would be not having to maintain a secondary list of your own content.
+
+Instead, Greenwood uses GraphQL + Apollo to make that a reality!  So instead of a static list, you can do something like this!
+
+```render javascript
+render() {
+  return html\`
+    <ul>
+      ${pages.map((page) => {
+        return html\`
+          <li><a href=\"${page.path}\">${page.title}</a></li>
+        \`;
+      })}
+    </ul>
+  \`;
+}
+```
+
+### Internal Sources
+Greenwood exposes a [GraphQL](https://graphql.org/) + [Apollo](https://www.apollographql.com/docs/apollo-server/) server locally when developing available at `localhost:4000` that can be used to get information about your local content like path, "slug", title and other useful information that will be dynamic to the content you have.  Programmatic access to this data can provide the oppourtunirty to share your content with your users in a way that supports sorting, filter, organizing, and more!
+
+![graphql-playground](/assets/graphql-playground.png)
+
+#### Schema
+To kick things off, let's review what is availalble to you from.  Currently, the main "API" is just a list of all pages in your _pages/_ directory, represented as a `Page` [type defintion](https://graphql.org/graphql-js/basic-types/).   This is called Greenwood's `graph`.
+
+This is that the schema looks like:
+```render javascript
+graph {
+  id, // (string) the unique ID given to the generated component as it's selector e.g. \`<wc-md-id></wc-md-id>\`
+
+  link,  // (string) A URL link, typically derived from the filesystem path, e.g. /blog/2019/first-post/ 
+  
+  title,  // (string) Useful for a page's <title> tag or the title attribute for an <a> tag, inferred from the filesystem path, e.g. "First Post"
+  
+  filePath, // (string) path to file
+
+  fileName, // (string) file name without extension/path, so that it can be copied to scratch dir with same name
+
+  template // (string) page template used for the page 
+}
+```
+
+> All queries return subsets and / or derivitives of the `graph`.
+
+#### Queries
+To help facilitiate development, Greenwood provides a couple queries out of the box that you can use to get access to the `graph` and start using it in your components, which we'll get to next.
+
+Below are the queries available:
+
+#### Graph
```

Comment:
Graph is a subheading of queries, but it shares the same font-size due to both being h4. What do you think about the organization here?

Reply:
I suppose I can just add one extra `#` to each heading then.  But maybe better to consider [`@data` as it's own package](https://github.com/ProjectEvergreen/greenwood/pull/269#pullrequestreview-347305483), as since it will be such a key feature, it might make sense to make it its own navigation item, which would give these docs more opportunity to breath, as it were.